src/duckling/project/map-select.component.ts

```diff
@@ -26,21 +27,20 @@ import {openDialog} from '../util/md-dialog';
                     </md-list-item>
```

Comment:
If you put the {{map}} within a div you can place `[title]="map"` on that div so there's a tooltip for when the title is truncated.